e2e/integration_tests/keras_to_tfjs_create_save_predict.js

```
@@ -0,0 +1,114 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+/**
+ * This file is 2/3 of the test suites for keras v3: create->save->predict.
+ * E2E from Keras model -> TFJS model -> Comparsion between two models.
+ *
+ * This file does below things:
+ * - Load the TFJS layers model to runtime.
+ * - Load inputs data.
+ * - Make inference and store in local files.
+ */
+const tf = require('@tensorflow/tfjs');
+const tfc = require('@tensorflow/tfjs-core');
+const tfconverter = require('@tensorflow/tfjs-converter');
+const tfl = require('@tensorflow/tfjs-layers');
+const tfjsNode = require('@tensorflow/tfjs-node');
+const ndarray = require('ndarray');
+const fs = require('fs');
+const join = require('path').join;
+const TEST_DATA_DIR = "keras_to_tfjs_create_save_predict_data";
+const _ = require("zlib")
+
+async function load_file(file_path, model_name) {
+
+  const p = tfjsNode.io.fileSystem(file_path)
+
+  const model = await tf.loadLayersModel("file://" + tfjsNode.io.fileSystem(file_path).path + "/model.json");
+
+  const xsDataPath = join(TEST_DATA_DIR, `${model_name}.xs-data.json`);
+  const xsShapesPath = join(TEST_DATA_DIR, `${model_name}.xs-shapes.json`);
+  const dataArray = fs.readFileSync(xsDataPath, "utf-8", (_, data) => {
+    const d = JSON.parse(data);
+    return d
+  });
+  const shapeArray = fs.readFileSync(xsShapesPath, "utf-8", (_, data) => {
+    const shape = JSON.parse(data);
+    return shape
+  });
+
+  const parsedShape = JSON.parse(shapeArray);
+  const parsedData = JSON.parse(dataArray);
+  const A = zip(parsedData, parsedShape);
+  const res = [];
+  for (let i = 0; i < A.length; i++) {
+    const value = A[i][0];
+    const shape = A[i][1];
+    res.push(ndarray(value, shape));
+  }
+  let z;
```

`z` is undefined for the next line if res.length is not 1. Should we throw an error in this case?